src/pages/SignUp.jsx

```diff
@@ -17,13 +15,11 @@ export default function SignUp() {
   // follow this function for a better practice with interactive button
 
   const { run: signUp, loading: signUpLoading } = useRequest(
-    // fixme: temp hard code company
+    // No harding for company required as it will be entered by user when registering for first time
 
     async (data) =>
       auth.signUp({
         ...data,
-        // fixme: temp hard code company
-        company: doc(database, "Companies", "KJLOQ9jWh9zsc3JfBugR"),
```

Comment:
we still need this part for employer user. The company has to be a document reference to make jobs work. we will work on fixing the company next sprint. 
please help to delete the `company` field below

Reply:
Added the temp hardcode company piece back and disabled the company field on the signup page